.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.16
+          go-version: ~1.16
```

Comment:
might as well bump to 1.17?

Reply:
will do in a follow up, just want to get deps merged first